src/style.css

```diff
@@ -1,3 +1,72 @@
+:root {
+  --primary: #00acdc;
+  --fnSmall: 12px;
+}
+
 body {
   margin: 0;
+  max-width: 1200px;
+  min-width: 1024px;
+  font-family: "Roboto", sans-serif;
+}
+
+.head {
```

Comment:
header should be the full body width
<img width="1202" alt="Screenshot 2023-10-04 at 20 39 19" src="https://github.com/mate-academy/layout_moyo-header/assets/117587463/11a061cf-ede8-4b29-8823-15b25ff7d358">